layouts/partials/footer.html

```diff
@@ -18,7 +18,9 @@
       </div>
       <div class="td-footer__copyright-etc col-12 col-sm-4 text-center py-2 order-sm-2">
         {{ with .Site.Params.copyright -}}
-          <span>&copy; {{ now.Year }} {{ . }} {{ T "footer_all_rights_reserved" }}</span>
+          <span>&copy; {{ with .year -}}{{- . -}}{{- else -}}{{- now.Year -}}{{- end }} {{ with .title -}}{{- . -}}{{- end }} {{ T "footer_all_rights_reserved" }}</span>
+        {{- else }}
+          {{ with .Site.Copyright }}<span>{{ . | safeHTML }}</span>{{ end }}
```

Comment:
Sorry, I shouldn’t have put a span.

```suggestion
          {{ with .Site.Copyright }}{{ . | safeHTML }}{{ end }}
```

Reply:
Removed the span as requested.